LayoutTests/media/media-h264-webm-error.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>media-vp8-webm-error</title>
+    <script src=video-test.js></script>
+    <script>
+        window.addEventListener('load', event => {
+        
+            run('video = document.createElement("video")');
+            run('document.body.appendChild(video)');
+            video.controls = 1;
+            video.muted = 1;
+
+            video.onerror = (event) => {
+                consoleWrite("received error event");
+                endTest();
```

Comment:
Nit: maybe `failTest` instead of `endTest`?

Ditto for the failure cases in the other tests.

Reply:
> Nit: maybe `failTest` instead of `endTest`?
> 
> Ditto for the failure cases in the other tests.

I use the different output between gstreamer player and AVF.
WPE and GTK will play those videos just fine (which is new). AVF doesn't.

So I set different expectations between platforms.